src/js/common/schemaform/FormPage.jsx

```diff
@@ -159,7 +159,7 @@ class FormPage extends React.Component {
   validate(formData, errors) {
     const { schema, uiSchema } = this.props.form[this.props.route.pageConfig.pageKey];
     if (uiSchema) {
-      uiSchemaValidate(errors, uiSchema, schema, formData);
+      uiSchemaValidate(errors, uiSchema, schema, schema.definitions, formData);
```

Comment:
We're passing the schema to validation functions now and we need to fill in any fields that use JSON Schema's common definitions.